docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,132 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+Here is a schematic diagram of this setup:
 
-_Here we provided a small sample on K8s_
+![postgresql-spring-boot](/img/docs/postgresql-spring-boot.png)
 
-## OpenShift
+The service binding operator will collect backing service configuration required
+for connectivity and expose it to the applications.  If service binding operator
```

Comment:
why do we speak hear about situation when SBO is installed.. this section is about installing and using SBO. We should at the end have some conversation around pro/cons of using SBO approach vs. non-SBO approach.

Reply:
Moved it to a summary section at the end. 74af4f09ee3f4810ffbfdb02fb498e9a91624a91